code/game/objects/objs.dm

```diff
@@ -1,36 +1,48 @@
 /obj
-	//var/datum/module/mod		//not used
-	var/origin_tech = null	//Used by R&D to determine what research bonuses it grants.
 	animate_movement = SLIDE_STEPS
-	var/list/species_exception = null	// list() of species types, if a species cannot put items in a certain slot, but species type is in list, it will be able to wear that item
-	var/sharp = FALSE		// whether this object cuts
-	var/in_use = FALSE // If we have a user using us, this will be set on. We will check if the user has stopped using us, and thus stop updating and LAGGING EVERYTHING!
+	/// Used by R&D to determine what research bonuses it grants.
+	var/origin_tech = null
+	/// `list()` that allows any contained species to wear the item.
```

Comment:
```suggestion
	/// A list of otherwise restricted /datum/species type paths that are permitted to wear this item.
```

Using the list constructor syntax to refer to the type feels off